components/inputs/input-styles.js

```diff
@@ -0,0 +1,72 @@
+import '../colors/colors.js';
+import { css } from 'lit-element/lit-element.js';
+
+export const inputStyles = css`
+	.d2l-input {
+		background-color: #ffffff;
+		border-color: var(--d2l-color-galena);
+		border-radius: 0.3rem;
+		border-style: solid;
+		border-width: 1px;
+		box-shadow: inset 0 2px 0 0 rgba(181, 189, 194, .2); /* corundum */
+		color: var(--d2l-color-ferrite);
+		box-sizing: border-box;
+		display: inline-block;
+		font-family: inherit;
+		font-size: 0.8rem;
+		font-weight: 400;
+		height: var(--d2l-input-height, auto);
+		letter-spacing: 0.02rem;
+		line-height: 1.2rem; /* using min-height AND line-height as IE11 doesn't support line-height on inputs */
+		margin: 0;
+		min-height: calc(2rem + 2px);
+		min-width: calc(2rem + 1em);
+		padding: var(--d2l-input-padding, 0.4rem 0.75rem);
+		position: relative;
+		vertical-align: middle;
+		width: 100%;
+		transition: background-color 0.5s ease, border-color 0.001s ease;
+	}
+	input.d2l-input::placeholder,
+	textarea.d2l-input::placeholder {
+		color: var(--d2l-input-placeholder-color, var(--d2l-color-mica));
+		font-size: 0.8rem;
+		font-weight: 400;
+		opacity: 1; /* Firefox has non-1 default */
+	}
```

Comment:
I ditched the pseudo elements for Firefox and Webkit since as of 2017 Chrome, Safari and Firefox all support the native `::placeholder`.